docs/source/projects.rst

```diff
@@ -0,0 +1,107 @@
+Projects with Myosuite
+#########################################
+
+.. _projects:
+
+* :ref:`myochallenge`
+    * :ref:`myo_challenge_22`
+    * :ref:`myo_challenge_23`
+    * :ref:`myo_challenge_24`
+* :ref:`pub_with_myosuite`
+    * :ref:`ref_myodex`
+    * :ref:`ref_deprl`
+    * :ref:`ref_lattic`
+    * :ref:`ref_sar`
+
+
+.. _myochallenge:
+
+MyoChallenge
+========================================
+The Myosuite Team organised MyoChallenge to tackle difficult problems in top-level machine learning conference.
+Our latest challenge is accepted to NeuIPs 2024.
+
+.. _myo_challenge_22:
+
+MyoChallenge-2022: Learning Physiological Dexterity
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+Introducing `MyoChallenge - a NeurIPS 2022 <https://sites.google.com/view/myochallenge>`__ competition track on learning contact-rich manipulation skills for a physiologically 
```

Comment:
@raku-slyu - this seems like a copy-paste. Harmonize langugae as per the content of the page.

Reply:
Rephrased the paragraph and removed the last line that is inappropriate here